### ISSUE TITLE
Add sorting to the Orchestra#show page

### ISSUE THREAD

**Issue by jmarsh24** [jmarsh24/tangocloud#362]
The current implementation sorts the Orchestra#show page by popularity score by default. I would like to be able to sort by year/title.

**Comment by eleonorapopovska** [comment]
On mobile, It would be nice to also not just sort, but  be able to directly type in and search for the Orchestra I am looking for. Sometimes, I need to scroll down too much to just find the Orchestra I already know I'm looking for.

**Comment by jmarsh24** [comment]
Isn't that also what the global search is for? We would be providing two search sections.

**Comment by eleonorapopovska** [comment]
Yes. However, when I'm already on the Orchestra Page, looking for an Orchestra, I have no need to navigate to the global Search Page.

**Comment by eleonorapopovska** [comment]
I know it is more an Android thing, but the function I had in mind is like a Local Search, maybe a floating Search Button. The truth is that search functionality especially on mobile is something I should be able to access very quick whenever I feel the need, and this also might be when I'm looking at a Playlist, or Orchestra Page. When I know I want to play Tinta Roja, while being on Troilo page, I just wanna be able to type in Tint... and get a result... And not go through all the filters, to get it. Many other platforms have a local search shortcut in some form everywhere, apart from the General Main Search/Browse Page. 

![IMG_0255](https://github.com/user-attachments/assets/aab2cca9-3b47-4461-b85a-2918bd1ecd77)
![IMG_0257](https://github.com/user-attachments/assets/f7cb57b6-2e23-4716-a245-f65a221de96f)
![IMG_0256](https://github.com/user-attachments/assets/d8580358-ec63-45cf-8ac9-7247d9bf73b1)

![2b46715be297244ea98893fc052f272c](https://github.com/user-attachments/assets/57564412-ebc4-41f6-8375-1f988b44e0dc)

We even had local Search ourselves once upon a time, but then we removed it as not essential.

<img width="1680" alt="Screenshot 2024-11-19 at 10 55 47" src="https://github.com/user-attachments/assets/8c48e7f4-ae3a-448f-8fe1-115a323c65fe">

**Comment by jmarsh24** [comment]
I think that I never use these localized searches and instead I search using the global search function, but we can discuss doing a localized search. I don't think it has anything to do with being a feature on android or iOS.

**Comment by eleonorapopovska** [comment]
As you've noted, many music players already include localized search, though implementation varies by platform. On Android, such functions are often placed in a floating action button at the bottom right for easy access, with context-specific functionality. While I initially imagined a floating search button, the key question, as you've pointed out, is whether we even need a localized search feature alongside the general search. Its final form—be it a button, search bar, or icon—can be decided later.
